thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/dashboard/views/diffsummary/Summary.java

```diff
@@ -186,7 +187,8 @@ private static boolean nodeIsThinnedOut(HierarchyNode node) {
   }
 
   private static void rollbackInsertions(HierarchyNode node, Set<HierarchyNode> answer, List<HierarchyNode> removedNodes) {
-    removedNodes.sort(NODE_COMPARATOR.reversed()); // Rollback from top to bottom nodes
+    Collections.sort(removedNodes, NODE_COMPARATOR); // Rollback from top to bottom nodes
+    Collections.reverse(removedNodes);
```

Comment:
Minor advise: Combine these two lines to Collections.sort(removedNodes, Collections.reverseOrder(NODE_COMPARATOR));